pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -118,6 +121,27 @@ let
     nativeBuildInputs = [ perl nukeReferences which libarchive ]
       ++ optionals (!libsOnly) kernel.moduleBuildDependencies;
 
+    env.libnvidia_nscq = let
```

Comment:
Wouldn't this make more sense as an additional package available through `passthru` like `nvidia-settings/persistenced` instead of copying it into the driver package?